crowbar_framework/app/models/node_object.rb

```diff
@@ -1207,7 +1207,7 @@ def run_ssh_cmd(cmd)
     args = ["sudo", "-i", "-u", "root", "--", "timeout", "-k", "5s", "15s",
             "ssh", "-o", "ConnectTimeout=10",
             "root@#{@node.name}",
-            cmd.to_s
+            "\"#{cmd}\""
```

Comment:
I would solve it this way: https://github.com/crowbar/crowbar-core/pull/623/commits/6d3014a096ac02ad298f8c3491aefe804e344f04

And this change was introduced to make commands like https://github.com/crowbar/crowbar-core/pull/623/files#diff-de2e101f791dc5ede34b391d8b632034R106 work. Maybe it's all wrong, it's still WIP